sdcm/nemesis.py

```diff
@@ -527,6 +527,23 @@ def disrupt_nodetool_enospc(self, sleep_time=30, all_nodes=False):
             nodes = [self.target_node]
         self._set_current_disruption('Enospc test on {}'.format([n.name for n in nodes]))
 
+        for node in nodes:
+            with DbEventsFilter(type='NO_SPACE_ERROR', node=node), \
+                    DbEventsFilter(type='BACKTRACE', line='No space left on device', node=node), \
+                    DbEventsFilter(type='DATABASE_ERROR', line='No space left on device', node=node), \
+                    DbEventsFilter(type='FILESYSTEM_ERROR', line='No space left on device', node=node):
+
+                result = node.remoter.run('cat /proc/mounts')
+                if '/var/lib/scylla' not in result.stdout:
+                    self.log.error("Scylla doesn't use an individual storage, skip enospc test")
+                    continue
+
+                self.reach_enospc_on_node(target_node=node, log_object=self.log)
+
+                self.clean_enospc_on_node(target_node=node, log_object=self.log, sleep_time=sleep_time)
+
+    @staticmethod
+    def reach_enospc_on_node(target_node, log_object):
         def search_database_enospc(node):
```

Comment:
If you're doing this, why not taking these two functions also out as staticmethods, so others will be able to use them independently.
(for example, checking that we didn't reach to enospc in certain cases).

Please ack and fix that in another PR (not a must for this one).

Reply:
You mean completely outside of the nemesis class? Sure I don't see why not, I'll create a pr after this one to change that

Reply:
No, I mean to take out def search_database_enospc from reach_enospc_on_node.

Reply:
Oh ok, yeah I agree, could be helpful, I'll do it in another pr

Reply:
I ended up doing it here